### ISSUE TITLE
MaxResultCount property not working

### ISSUE THREAD

**Issue by kforshier** [Azure/azure-spatial-anchors-samples#93]
Using the Android SDK 2.0.0, Pixel 2 device on Android 10

My assumption is that when creating a near device criteria, setting the maxResultCount property would limit the number of anchors that are located. However in my testing, I have the maxResultCount set to a low number, such as 5, but am still getting more than 5 anchors located. Sample code is below. Is this a bug or am I misunderstanding how that property should work?
```
        val criteria = AnchorLocateCriteria()

        val nearDeviceCriteria = NearDeviceCriteria()
        nearDeviceCriteria.distanceInMeters = 150f
        nearDeviceCriteria.maxResultCount = 5

        criteria.nearDevice = nearDeviceCriteria

        stopWatcher()

        cloudAnchorManager?.startLocating(criteria)
```

**Comment by weinanshen** [comment]
Hi @kforshier, it sounds like a bug, we'll investigate it and get back to you. Thanks!

**Comment by kforshier** [comment]
Just to add, I tested this on the iOS SDK and it was not working either

**Comment by weinanshen** [comment]
Hi @kforshier, this is a bug in SDK and we're working on a fix for it. In the meanwhile, as a workaround, you will be getting back all the results and can do custom filtering on your end until we fix this.

**Comment by TobiasPott** [comment]
Is there any update on this and was this resolved since SDK 2.0.0?
It is not mentioned in any of the last releases since then.

**Comment by craigktreasure** [comment]
This was fixed in 2.6.0.
